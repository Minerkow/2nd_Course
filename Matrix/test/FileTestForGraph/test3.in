1 -- 2, 2.0; -12.0
2 -- 3, 2.0;
3 -- 1, 2.0; -12.0
1 -- 1, 10.0; 10.0