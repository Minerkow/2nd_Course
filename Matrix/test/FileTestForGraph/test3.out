1 -- 2: -4 A
2 -- 3: -4 A
3 -- 1: -4 A
1 -- 1: 0 A